This breaks our multilib builds:

  Confirm gpg-error-config works... no
  ***  Please report to <https://bugs.gnupg.org> with gpg-error-config-test.log

--- libgpg-error-1.44/src/Makefile.am
+++ libgpg-error-1.44/src/Makefile.am
@@ -347,14 +347,6 @@
 	cp gpg-error.h gpgrt.h
 
 gpg-error-config: gpgrt-config gpg-error-config-old gpg-error-config-test.sh
-	@echo $(ECHO_N) "Confirm gpg-error-config works... $(ECHO_C)"
-	@if ./gpg-error-config-test.sh --old-new; then \
-	  echo "good";                                         \
-	else                                                   \
-	  echo "no";                                           \
-	  echo "***  Please report to <https://bugs.gnupg.org> with gpg-error-config-test.log"; \
-	  exit 1; \
-	fi
 	cp gpg-error-config-old $@
 
 install-data-local:
